jobs/haproxy/spec

```diff
@@ -114,3 +117,18 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
```

Comment:
697719dac8b2a26261f2aab8a6d4817bf68cda0d

default set to 1, 0 is refused by keepalived